spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,35 @@
+feature 'viewing a peep' do
+  scenario 'peeps should display user and user_name' do
+    sign_up
+    peep
+    within 'ul#peeps' do
+      expect(page).to have_content 'Edward Kerry (Edward) peeped: This is Ed\'s peep'
+    end
+  end
+
+  scenario 'peeps should appear in reverse order do' do
+    sign_up
+    Timecop.freeze do
+    3.times do |peep|
+      click_button "Compose a peep!"
+      fill_in :peep, with: "#{peep + 1}"
+      click_button 'Peep!'
+    end
+    within 'ul#peeps' do
+      expect(page).to have_content "Edward Kerry (Edward) peeped: 3 #{Time.now.strftime('%H:%M, %e %B %Y')}
```

Comment:
Metrics/LineLength: Line is too long. [107/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)